tests/Propel/Tests/Generator/Behavior/I18n/I18nBehaviorObjectBuilderModifierTest.php

```diff
@@ -31,6 +31,11 @@
  */
 class I18nBehaviorObjectBuilderModifierTest extends TestCase
 {
+    /**
+     * @var \Propel\Runtime\Connection\ConnectionWrapper 
+     */
+    protected $con;
```

Comment:
It'd be better to start fully typing class properties instead of using phpdoc.  With PHP 8, most phpdoc can be deleted

Reply:
Thank you for having a look and thank you for the reminder! I keep forgetting that PHP supports this now too.
I'll keep it in mind for changes in src!